core/config/docs/chains-evm.toml

```
@@ -160,9 +160,9 @@ PriceMax = '115792089237316195423570985008687907853269984665.6405640394575840079
 PriceMin = '1 gwei' # Default
 # LimitDefault sets default gas limit for outgoing transactions. This should not need to be changed in most cases.
 # Some job types, such as Keeper jobs, might set their own gas limit unrelated to this value.
-LimitDefault = 500_000 # Default
+LimitDefault = 8000000 # Default
```

Let's retain the format for readability:
```suggestion
LimitDefault = 8_000_000 # Default
```